bot/.env.example

```
@@ -0,0 +1,16 @@
+SPOT_STREAM_URL=wss://testnet.binance.vision/ws
+FUT_API_URL=https://testnet.binancefuture.com
+FUT_WSS_URL=wss://testnet.binancefuture.com/ws-fapi/v1
+SPOT_API_KEY=
+SPOT_API_SECRET=
+FUT_API_KEY=
+FUT_API_SECRET=
+BASE_USDT=50
+MIN_EDGE_BPS=8
+TAKER_FEE_SPOT=0.001
+TAKER_FEE_FUT=0.0004
+SYMBOLS= BTCUSDT,ETHESUST
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)